doc/release/RELEASE-NOTES.md

```diff
@@ -29,6 +29,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * [OSDEV-1777](https://opensupplyhub.atlassian.net/browse/OSDEV-1777) - A consistent URL style was established across all pages of the SLC workflow. After the changes, the URL begins from `/contribute/single-location/`.
 * [OSDEV-1678](https://opensupplyhub.atlassian.net/browse/OSDEV-1678) - Added asterisks next to each required form field (Name, Address, and Country) on the "Production Location Information" page. Highlighted an empty field and displayed an error message if it loses focus.
 * [OSDEV-1778](https://opensupplyhub.atlassian.net/browse/OSDEV-1778) - Fixed the validation for number of workers field in POST, PATCH production locations API. The min field must be less than or equal to the max field.
+* [OSDEV-1806](https://opensupplyhub.atlassian.net/browse/OSDEV-1806) - Refactored Parent Company field validation.
```

Comment:
Sorry, I missed.
Vadim, could you please move your note to release 1.32 in the RELEASE-NOTES.md file? It looks like it won’t be released in 1.31.

Reply:
@vladsha-dev 
> move your note to release 1.32

Maybe it will be releae 2.0.0 then?

Reply:
I wouldn't change it in your PR. It should be done in this PR: https://github.com/opensupplyhub/open-supply-hub/pull/535.

Reply:
Your changes don’t initiate 2.0.0.

Reply:
Applied change of release notes.